src/views/Settings.vue

```diff
@@ -8,52 +8,102 @@
     </ion-header>
     
     <ion-content>
-       <!-- Select eCom store -->
-      <ion-item>
-        <ion-icon :icon="globeOutline" slot="start" />
-        <ion-label>{{$t("eCom Store")}}</ion-label>
-        <ion-select interface="popover" :value="currentEComStore.productStoreId" @ionChange="setEComStore($event)">
-          <ion-select-option v-for="store in (userProfile ? userProfile.stores : [])" :key="store.productStoreId" :value="store.productStoreId" >{{ store.storeName }}</ion-select-option>
-        </ion-select>
-      </ion-item>
-
-      <!-- OMS information -->
-      <ion-item>
-        <ion-icon :icon="codeWorkingOutline" slot="start"/>
-        <ion-label>{{ $t("OMS") }}</ion-label>
-        <p slot="end">{{ baseURL ? baseURL : instanceUrl }}</p>
-      </ion-item>
-
-      <!-- User timezone -->
-      <ion-item>
-        <ion-icon :icon="timeOutline" slot="start"/>
-        <ion-label> {{ userProfile && userProfile.userTimeZone ? userProfile.userTimeZone : '-' }} </ion-label>
-        <ion-button @click="changeTimeZone()" slot="end" fill="outline" color="dark">{{ $t("Change") }}</ion-button>
-      </ion-item>
-
-      <!-- Profile of user logged in -->
-      <ion-item>
-        <ion-icon :icon="personCircleOutline" slot="start" />
-        <ion-label>{{ userProfile !== null ? userProfile.partyName : '' }}</ion-label>
-        <ion-button slot="end" fill="outline" color="dark" @click="logout()">{{ $t("Logout") }}</ion-button>
-      </ion-item>
-
+      <div class="user-profile">
+        <ion-card>
+          <ion-item lines="full">
+            <ion-avatar slot="start" v-if="userProfile?.partyImageUrl">
+              <Image :src="userProfile.partyImageUrl"/>
+            </ion-avatar>
+            <ion-label>
+              {{ userProfile?.partyName }}
+              <p>{{ userProfile?.userLoginId }}</p>
+            </ion-label>
+          </ion-item>
+          <ion-button fill="outline" color="danger" @click="logout()">{{ $t("Logout") }}</ion-button>
+          <!-- Commenting this code as we currently do not have reset password functionality -->
+          <!-- <ion-button fill="outline" color="medium">{{ $t("Reset password") }}</ion-button> -->
+        </ion-card>
+      </div>
+      <h1>{{ $t('OMS') }}</h1>
+      <section>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-subtitle>
+              {{ $t("OMS instance") }}
+            </ion-card-subtitle>
+            <ion-card-title>
+              {{ instanceUrl }}
```

Comment:
This is changed.
https://github.com/hotwax/threshold-management/pull/147/files#diff-817427b0a52f22c1023629bd0728cd3102a9dd01cb780ef55f79d99ec24c23e6L24

Check for other code as well

Reply:
Ok sir